packages/core/src/components/Button/button.module.css

```diff
@@ -17,6 +19,18 @@
 .cds-button--tertiary:not([disabled]):active {
   box-shadow: none;
 }
+.cds-button--reverseTextIcon {
```

Comment:
Thoughts on replacing camelCasing with kebab-cased-class-names (lowercase + dash-separated)? I don’t feel strongly as long as we’re consistent with one approach.

Reply:
You're right, we should stick to kebab, since we're already using it in places. I'll update.